farmhelden/rest_server/serializers.py

```diff
@@ -161,19 +161,22 @@ def validate(self, data):
         }
 
 
-class CampaignSerializer(serializers.HyperlinkedModelSerializer):
+class LocationSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
-        model = Campaign
-        fields = ['id', 'farm_id', 'location_id', 'date_from', 'date_to']
+        model = Location
+        fields = ['id', 'point', 'info', 'farm_id', 'location_type']
 
     def create(self, validated_data):
-        return Campaign.objects.create(**validated_data)
+        return Location.objects.create(**validated_data)
 
 
-class LocationSerializer(serializers.HyperlinkedModelSerializer):
+class CampaignSerializer(serializers.HyperlinkedModelSerializer):
+    location_id = LocationSerializer(read_only=True)
```

Comment:
Hier müsste dann nur location sein oder? Und entsprechend in den fields ein weiters Feld mit location?

Reply:
ah ja, sonst sieht die response komishc aus, ich ändere das.